addons/account_analytic_default_hr_expense/models/hr_expense.py

```diff
@@ -7,10 +7,8 @@
 class HrExpense(models.Model):
     _inherit = 'hr.expense'
 
-    @api.onchange('product_id', 'date')
-    def _onchange_product_id(self):
-        res = super(HrExpense, self)._onchange_product_id()
-        rec = self.env['account.analytic.default'].account_get(self.product_id.id, company_id=self.company_id.id, date=self.date)
-        self.analytic_account_id = rec.analytic_id.id
-        self.analytic_tag_ids = rec.analytic_tag_ids.ids
-        return res
+    @api.onchange('product_id', 'date', 'account_id')
```

Comment:
why do you need a dependency to `account_id` ? It doesn't seem used....

Reply:
Right. Not needed in 12.3. Will be needed for 13.0 & up

Reply:
Removed. Will need to be added again when forwardported to 13.0

Reply:
you can already make the PR for 13.0